superset-frontend/src/views/CRUD/data/database/DatabaseModal/index.tsx

```diff
@@ -299,6 +301,73 @@ const DatabaseModal: FunctionComponent<DatabaseModalProps> = ({
     }
   };
 
+  const setDatabaseModel = option => {
```

Comment:
can the param be "engine" if it's just a string? Maybe a type on that would help, too.